src/send-linux.h

```diff
@@ -5,12 +5,20 @@
  * use this file except in compliance with the License. You may obtain a copy
  * of the License at http://www.apache.org/licenses/LICENSE-2.0
  */
-
 #ifndef ZMAP_SEND_LINUX_H
 #define ZMAP_SEND_LINUX_H
 
-#include "../lib/includes.h"
+#include <stdlib.h>
+#include <stdio.h>
+#include <netinet/ip.h>
```

Comment:
FWIW, I think we try to put the Linux headers after the C stdlib, but we're probably not consistent about it.

C stdlib -> Linux/System Headers -> ZMap headers